### PR TITLE
En 935 fix data models endpoints

### PR DESCRIPTION
**Pull request by eugeniosant**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java [hunk 1]
```diff
@@ -76,8 +78,17 @@ public void setDataObjectModelValidator(DataObjectModelValidator dataObjectModel
 
     @RestAccessControl(permission = Permission.SUPERUSER)
     @RequestMapping(value = "/{dataModelId}", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<?> getDataObjectModel(@Valid @PathVariable Long dataModelId) {
-        DataModelDto dataModelDto = this.getDataObjectModelService().getDataObjectModel(dataModelId);
+    public ResponseEntity<?> getDataObjectModel(@PathVariable String dataModelId) {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java]: Add some debug logging for controller methods

File: engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java [hunk 2]
```diff
@@ -91,15 +102,23 @@ public void setDataObjectModelValidator(DataObjectModelValidator dataObjectModel
...
+            }
         }
         DataModelDto dataModelDto = this.getDataObjectModelService().addDataObjectModel(dataObjectModelRequest);
         return new ResponseEntity<>(new RestResponse(dataModelDto), HttpStatus.OK);
     }
 
     @RestAccessControl(permission = Permission.SUPERUSER)
     @RequestMapping(value = "/{dataModelId}", method = RequestMethod.PUT, produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<?> updateGroup(@Valid @PathVariable Long dataModelId, @Valid @RequestBody DataObjectModelRequest dataObjectModelRequest, BindingResult bindingResult) {
+    public ResponseEntity<?> updateGroup(@PathVariable String dataModelId, @Valid @RequestBody DataObjectModelRequest dataObjectModelRequest, BindingResult bindingResult) {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java]: Debug logging for controller methods

File: engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java [hunk 3]
```diff
@@ -108,15 +127,32 @@ public void setDataObjectModelValidator(DataObjectModelValidator dataObjectModel
...
+            }
+        }
         DataModelDto dataModelDto = this.getDataObjectModelService().updateDataObjectModel(dataObjectModelRequest);
         return new ResponseEntity<>(new RestResponse(dataModelDto), HttpStatus.OK);
     }
 
     @RestAccessControl(permission = Permission.SUPERUSER)
     @RequestMapping(value = "/{dataModelId}", method = RequestMethod.DELETE, produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<?> deleteGroup(@Valid @PathVariable Long dataModelId) throws ApsSystemException {
+    public ResponseEntity<?> deleteGroup(@PathVariable String dataModelId) throws ApsSystemException {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java]: Debug logging for controller methods

File: engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java (hunk 1 above)
Comment (eugeniosant) [reply engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java]: Done

File: engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java (hunk 2 above)
Comment (eugeniosant) [reply engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java]: Done

File: engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java (hunk 3 above)
Comment (eugeniosant) [reply engine/src/main/java/org/entando/entando/web/dataobjectmodel/DataObjectModelController.java]: Done

**Pull request merged**